aruco_opencv/src/aruco_tracker.cpp

```diff
@@ -408,6 +427,13 @@ class ArucoTracker : public rclcpp_lifecycle::LifecycleNode
     }
   }
 
+  void callback_compressed_image(const sensor_msgs::msg::CompressedImage::ConstSharedPtr img_msg)
+  {
+    // Decompress the image using bridge
+    auto image = cv_bridge::toCvCopy(img_msg, "bgr8");
+    callback_image(image->toImageMsg());
```

Comment:
Attention: I just looked through the code again, and I noticed that this will not preserve the header of the message.